databricks/sdk/credentials_provider.py

```diff
@@ -601,6 +658,19 @@ def __init__(self) -> None:
     def auth_type(self) -> str:
         return self._auth_type
 
+    def oauth_token(self, cfg: 'Config') -> Token:
+        auth_providers = [
+            pat_auth, basic_auth, metadata_service, oauth_service_principal, azure_service_principal,
+            github_oidc_azure, azure_cli, external_browser, databricks_cli, runtime_native_auth,
+            google_credentials, google_id
+        ]
```

Comment:
Let's just initialize this in the `__init__` constructor rather than copying it.